lib/trento.pm

```diff
@@ -767,12 +767,13 @@ Get the api-key from the Trento installation
 =cut
 
 sub trento_api_key {
-    my ($wd, $basedir) = @_;
-    my $cmd = join(' ', $basedir . '/trento-server-api-key.sh',
+    my ($basedir) = @_;
```

Comment:
if you remove `$wd` from here (that is fine) you also need to remove it from the caller https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/25bec175868356be70cde4d3796c2a02fd4f4256/tests/sles4sap/trento/install_agents.pm#L23

Reply:
It should still work, but I agree that it's better to remove it from the caller as well.